abcclassroom/example-data/sample_config.yml

```diff
@@ -0,0 +1,56 @@
+
+# The GitHub organization to host your GitHub classroom
+# Create a new organization on GitHub and change this to the name you gave it
+organization: earth-analytics-edu
+
+# (Optional) Short name for organization. If set, this gets used to prepend
+# the repositories instead of the full organization name, just to make
+# names shorter and less unwieldy
+organization-shortname: ea
```

Comment:
similarly this should be populated with something more generic. i am not sure how this is used yet so let's talk more.

@kcranston how is the shortname used in this workflow? i'm not sure what it's used for or why we need it.

Reply:
this is used as the acronym as per the naming conventions (https://hackmd.io/_4GDiG9wSwe6lBq18bDM7w?view#Naming-conventions).